README.md

```diff
@@ -218,7 +297,7 @@ credentials:
   # Default: No server.
   # May be overridden from the command-line with parameter `--server`.
 
-# Optional
+# --- Worker configuration. Optional. CURRENTLY NOT MAINTAINED.
```

Comment:
Hmm? What happened?

Reply:
Well, a few months ago, after I spent weeks working on it, something in Synapse changed and broke workers. I haven't had the courage to troubleshoot this.